Sources/Rego/Engine.swift

```diff
@@ -104,21 +158,67 @@ extension OPA.Engine {
             try await store.write(to: StoreKeyPath(["data"]), value: bundle.data)
         }
 
+        let evaluator: IREvaluator
+
         if self.policies.count > 0 {
             guard loadedBundles.isEmpty else {
                 throw RegoError.init(code: .invalidArgumentError, message: "Cannot mix direct IR policies with bundles")
             }
-            return PreparedQuery(
-                query: query,
-                evaluator: IREvaluator(policies: self.policies),
-                store: self.store
-            )
+
+            evaluator = IREvaluator(policies: self.policies)
+        } else {
+            evaluator = try IREvaluator(bundles: loadedBundles)
+        }
+
+        // Load the capabilities
```

Comment:
⛏️ should we refactor the capabilities handling here out into some kind of `verifyCapabilities` method?